### DISCUSSION TITLE
Is this library compatible with Java?

### DISCUSSION THREAD

**Discussion by louchiiin** [patrykandpatrick/vico#299] [Questions] [answered]
Can I use this library in android studio (java)

**Comment by patrickmichalik** [comment, marked as answer]
Hello! You can add Vico to any Android project, whatever IDE you use. Being written in Kotlin, Vico is, as such, compatible with Java, but it’s not written with Java in mind. Functions with default arguments aren’t annotated with `@JvmOverloads`, for example. Because Jetpack Compose requires Kotlin, if you were to use Vico with Java, you would, of course, be restricted to the view version.
